Libplanet/Net/Swarm.cs

```diff
@@ -892,6 +1045,60 @@ private void BroadcastTxIds(IEnumerable<TxId> txIds)
                         break;
                     }
 
+                case GetRecentStates getRecentStates:
+                    {
+                        HashDigest<SHA256> blockHash = getRecentStates.BlockHash;
+                        IImmutableDictionary<HashDigest<SHA256>,
+                            IImmutableDictionary<Address, object>
+                        > blockStates = null;
+                        IImmutableDictionary<Address, IImmutableList<HashDigest<SHA256>>>
+                            stateRefs = null;
+                        IImmutableDictionary<Address, long> txNonces = null;
+
+                        if (_blockChain.Blocks.ContainsKey(blockHash))
+                        {
+                            // FIXME: Swarm should not directly access to the IStore instance,
+                            // but BlockChain<T> should have an indirect interface to its underlying
+                            // store.
+                            IStore store = _blockChain.Store;
+                            string ns = _blockChain.Id.ToString();
+
+                            IImmutableSet<Address> addresses =
```

Comment:
Do we need this variable? It is only used when creating `stateRefs` and seems to be no longer used.